t/apicast-async-reporting.t

```diff
@@ -1,202 +1,181 @@
+iWhvvvYDltIzsD75vqNQME4wHQYDVR0OBBYEFOBBS7ZF8Km2wGuLNoXFAcj0Tz1D
+MB8GA1UdIwQYMBaAFOBBS7ZF8Km2wGuLNoXFAcj0Tz1DMAwGA1UdEwQFMAMBAf8w
+CgYIKoZIzj0EAwIDRwAwRAIgZ54vooA5Eb91XmhsIBbp12u7cg1qYXNuSh8zih2g
+QWUCIGTHhoBXUzsEbVh302fg7bfRKPCi/mcPfpFICwrmoooh
 -----END CERTIFICATE-----
 >>> server.key
 -----BEGIN EC PARAMETERS-----
 BggqhkjOPQMBBw==
 -----END EC PARAMETERS-----
 -----BEGIN EC PRIVATE KEY-----
-MHcCAQEEIHIYTPgt2XlDTuL6Ly1jIqhlhM3lEspTyVldsaAoaC54oAoGCCqGSM49
-AwEHoUQDQgAEBmPcgjjxOmODfbQGqjkVtbq6qvFC8t0A5FWnL3nRQjI5nB9k7tX7
-vTx1NzFzq+w3Vf3vX+FqsWLyaBIhDSyUHg==
+MHcCAQEEIFCV3VwLEFKz9+yTR5vzonmLPYO/fUvZiMVU1Hb11nN8oAoGCCqGSM49
+AwEHoUQDQgAEhkmo6Xp/9W9cGaoGFU7TaBFXOUkZxYbGXQfxyZZucIQPt89+4r1c
+bx0wVEzbYK5wRb7UiWhvvvYDltIzsD75vg==
 -----END EC PRIVATE KEY-----
---- request
-GET /test?user_key=foo
 --- no_error_log
 [error]
---- response_body
-api response
---- error_code: 200
---- wait: 3
+
 
 === TEST 3: uses endpoint host as Host header
 when connecting to the backend
---- main_config
-env RESOLVER=127.0.0.1:$TEST_NGINX_RANDOM_PORT;
---- http_config
-include $TEST_NGINX_UPSTREAM_CONFIG;
-lua_package_path "$TEST_NGINX_LUA_PATH";
-lua_shared_dict api_keys 1m;
-init_by_lua_block {
-  ngx.shared.api_keys:set('42:val:usage%5Bhits%5D=2', 200)
-  require('apicast.configuration_loader').mock({
-    services = {
+--- configuration
+{
+    "services": [
       {
-        id = 42,
-        backend_version = 1,
-        backend_authentication_type = 'service_token',
-        backend_authentication_value = 'service-token',
-        proxy = {
-          api_backend = "http://127.0.0.1:$TEST_NGINX_SERVER_PORT/api/",
-          backend = {
-            endpoint = 'http://localhost.example.com:$TEST_NGINX_SERVER_PORT'
-          },
-          proxy_rules = {
-            { pattern = '/', http_method = 'GET', metric_system_name = 'hits', delta = 2 }
-          }
+        "id": 42,
+        "backend_version": 1,
+        "backend_authentication_type": "service_token",
+        "backend_authentication_value": "token-value",
+        "proxy": {
+            "api_backend": "http://test:$TEST_NGINX_SERVER_PORT/api-backend/",
+            "proxy_rules": [
+                { "pattern" : "/", "http_method" : "GET", "metric_system_name" : "hits", "delta" : 2 }
+            ]
         }
-      },
-    }
-  })
-}
---- config
-include $TEST_NGINX_APICAST_CONFIG;
-
-location /api/ {
-  echo "all ok";
+      }
+    ]
 }
-
+--- backend
 location /transactions/authrep.xml {
   content_by_lua_block {
-    if ngx.var.host == 'localhost.example.com' then
-      ngx.exit(200)
-    else
-      ngx.log(ngx.ERR, 'invalid host: ', ngx.var.host)
-      ngx.exit(404)
-    end
+      if ngx.var.host == 'test_backend' then
+        ngx.exit(200)
+      else
+        ngx.log(ngx.ERR, 'invalid host: ', ngx.var.host)
+        ngx.exit(404)
+      end
   }
 }
-
---- request
-GET /t?user_key=val
---- response_body
-all ok
---- error_code: 200
---- udp_listen random_port env chomp
```

Comment:
not sure this is feasible with Blackbox but in my opinion not needed since we have `test_backend` as a default hostname that we can check